experimenter/experimenter/nimbus_ui/templates/common/filter_dropdown.html

```diff
@@ -0,0 +1,64 @@
+{# Reusable filter dropdown for checkboxes with optional icons #}
+{% load nimbus_extras %}
+
+<div class="dropdown dropup ms-1 position-static">
+  <button class="btn btn-sm p-0 border-0 bg-transparent"
+          type="button"
+          id="{{ filter_name }}FilterDropdown-{{ dropdown_id|default:'0' }}"
+          data-bs-toggle="dropdown"
+          aria-expanded="false"
+          title="Filter by {{ filter_label }}">
+    <i class="fa-solid fa-filter text-body"></i>
+  </button>
+  <div class="dropdown-menu p-2 shadow-sm border rounded-2"
+       style="min-width:180px;
+              max-height:50vh;
+              overflow:auto">
+    <strong class="dropdown-header">Filter by {{ filter_label }}</strong>
+    <form method="get"
+          hx-get="{{ url }}"
+          hx-trigger="change"
+          hx-target="{{ hx_target }}"
+          hx-select="{{ hx_select }}"
+          hx-swap="outerHTML"
+          class="m-0 {{ filter_name }}-filter">
+      {% for form_field in form %}
+        {% if form_field.name != filter_name and form_field.name != "sort" %}
+          {% if form_field.value %}{{ form_field.as_hidden }}{% endif %}
+        {% endif %}
+      {% endfor %}
+      {% if current_sort %}<input type="hidden" name="sort" value="{{ current_sort }}">{% endif %}
+      <input type="hidden" name="my_deliveries_page" value="1">
+      {% for value, label in choices %}
+        {% if icon_filter == "qa_icon_info" %}
+          {% with icon_info=value|qa_icon_info %}
+            <div class="form-check">
+              <input class="form-check-input"
+                     type="checkbox"
+                     name="{{ filter_name }}"
+                     id="{{ filter_name }}-{{ forloop.counter }}"
```

Comment:
suggestion(not-blocking):

Could this be 
```suggestion
                     id="{{ filter_name }}-{{ value }}"
```